examples/training/train_morphologizer.py

```diff
@@ -84,8 +85,10 @@ def main(lang="en", output_dir=None, n_iter=25):
     morphologizer = nlp.create_pipe("morphologizer")
     nlp.add_pipe(morphologizer)
 
-    # add labels
-    for _, annotations in TRAIN_DATA:
+    # add labels and create the Example instances
+    train_examples = []
+    for text, annotations in TRAIN_DATA:
+        train_examples.append(Example.from_dict(nlp.make_doc(text), annotations))
```

Comment:
It's probably better to have the loop fetch the tags and morphology out of the `Example.reference` `Doc` object, so that it's easier to use the example on converted data.

I don't want to request changes here though, especially since we'll be changing the examples more later anyway and removing a bunch.

Reply:
Yea I just pretty much did a ctrl-F sweep and replaced whatever I found, but yea some of these examples are probably redundant in v.3.